xml/runtime_config.xml

```diff
@@ -294,11 +294,101 @@
    </sect3>
   </sect2>
  </sect1>
- <sect1 xml:id="ceph-config-key-store">
+ <sect1 xml:id="cha-ceph-config-key-store">
   <title>Config-key Store</title>
   <para>
-   Config-key store is a general purpose key/value service offered by the monitors.
+   Config-key is a general purpose service offered by the &mon;s. It simplifies
```

Comment:
Is `config-key` a command, or a program? I think when it is first introduced, it should be emphasized in some way, e.g. [systemitem class=service] or something.

Reply:
@lproven made it an entity, have a look once more please